components/org.wso2.carbon.identity.oauth2.grant.organizationswitch/src/main/java/org/wso2/carbon/identity/oauth2/grant/organizationswitch/internal/OrganizationSwitchGrantServiceComponent.java

```diff
@@ -28,5 +34,39 @@
         immediate = true
 )
 public class OrganizationSwitchGrantServiceComponent {
-    
+
+    private static final Log LOG = LogFactory.getLog(OrganizationSwitchGrantServiceComponent.class);
+
+    /**
+     * Set realm service implementation.
+     *
+     * @param realmService RealmService
+     */
+    @Reference(
+            name = "user.realmservice.default",
+            service = RealmService.class,
+            cardinality = ReferenceCardinality.MANDATORY,
+            policy = ReferencePolicy.DYNAMIC,
+            unbind = "unsetRealmService"
+    )
+    protected void setRealmService(RealmService realmService) {
+
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Setting the Realm Service.");
+        }
+        OrganizationSwitchGrantDataHolder.getInstance().setRealmService(realmService);
+    }
+
+    /**
+     * Unset realm service implementation.
+     *
+     * @param realmService RealmService
+     */
+    protected void unsetRealmService(RealmService realmService) {
+
+        if (LOG.isDebugEnabled()) {
```

Comment:
No need of `LOG.isDebugEnabled()` check here right? Check other place as well

Reply:
Fixed by - https://github.com/wso2-extensions/identity-oauth2-grant-organization-switch/pull/8/commits/51cdb1894ddca3f2acad76138c24c337b67be40d